web/src/main/java/org/mskcc/cbio/oncokb/api/pub/v1/UtilsApiController.java

```diff
@@ -263,7 +263,7 @@ public ResponseEntity<String> utilsAllActionableVariantsTxtGet(
         return new ResponseEntity<>(sb.toString(), HttpStatus.OK);
     }
 
-    private List<ActionableGene> getAllActionableVariants(Boolean isTextFile) {
+    public List<ActionableGene> getAllActionableVariants(Boolean isTextFile) {
```

Comment:
Is there a way to refactor this into a utility method? I think having other controllers call each other will be a little confusing since people won't be expecting it.